vcs/src/main/java/org/openjdk/skara/vcs/tools/PatchHeader.java

```diff
@@ -103,6 +103,9 @@ public static PatchHeader fromRawLine(String line) {
             sourcePath = Path.of(parts[1]);
         } else if (status.isUnmerged()) {
             sourcePath = Path.of(parts[1]);
+        } else if (status.isFileTypeChanged()) {
+            sourcePath = Path.of(parts[1]);
+            targetPath = sourcePath;
```

Comment:
I am not very sure whether this is right or not. I could not create a Status T raw line by myself